src/graphql/main/schema.graphql

```diff
@@ -1005,6 +1036,27 @@ type QuizQuestion {
   id: ID!
 }
 
+type RealtimePrice {
+  btcSatPrice: PriceOfOneSat!
+  denominatorCurrency: DisplayCurrency!
+  id: ID!
+
+  """
+  Unix timestamp (number of seconds elapsed since January 1, 1970 00:00:00 UTC)
+  """
+  timestamp: Timestamp!
+  usdCentPrice: PriceOfOneUsdCent!
+}
+
+input RealtimePriceInput {
+  currency: DisplayCurrency = "USD"
+}
+
+type RealtimePricePayload {
+  errors: [Error!]!
```

Comment:
why does this have an error array? isn't that the pattern for mutations only?

Reply:
this is part of subscriptions too because you dont want to throw and close the connection but you want to inform the user that we had an issue (specially with subscriptions like myUpdates)

Reply:
ok I see the error in `MyUpdatesPayload`.

we're not using this at all in the mobile repo (another example of something  exposed to the mobile that is not been used).

What action would you suggest on `errors`? 

it's a string/unstructured object so not very convenient to parse